### DISCUSSION TITLE
How do I create a bulk redeem command for multiple accounts?

### DISCUSSION THREAD

**Discussion by randomnew24** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#338] [Q&A]
I've seen that some people have made a "bulk redeem" command, allowing them to redeem a code across multiple accounts simultaneously. I'd like to know how this is done?

**Comment by LeleDerGrasshalmi** [comment]
you dont, we do not spoonfeed here. there is no (public) bulk redeem endpoint.
